newIDE/app/src/InstancesEditor/index.js

```diff
@@ -283,6 +287,7 @@ export default class InstancesEditorContainer extends Component {
       this.grid.setOptions(nextProps.options);
       this.instancesMover.setOptions(nextProps.options);
       this.instancesResizer.setOptions(nextProps.options);
+      this.instancesRotator.setOptions(nextProps.options);
```

Comment:
Sorry, there is the line to remove too, otherwise I guess it will crash

Reply:
I removed it already :)
https://github.com/4ian/GDevelop/pull/691/commits/8d6f5604e58852f50e5e5d351863dc26d5bf51c8